.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile

```diff
@@ -134,14 +134,25 @@ pipeline {
                         }
                     }
                 }
-                stage("7.12.0") {
+                stage("7.12.1") {
                     agent {
                         label 'linux'
                     }
                     steps {
                         unstash "source"
                         script {
-                            runWith(lib, failedTests, "eck-712-${BUILD_NUMBER}-e2e", "7.12.0")
+                            runWith(lib, failedTests, "eck-712-${BUILD_NUMBER}-e2e", "7.12.1")
+                        }
+                    }
+                }
+                stage("7.13.0") {
+                    agent {
+                        label 'linux'
+                    }
+                    steps {
+                        unstash "source"
+                        script {
+                            runWith(lib, failedTests, "eck-713-${BUILD_NUMBER}-e2e", "7.13.0")
```

Comment:
The declaration to delete the cluster is missing:
```diff
diff --git a/.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile b/.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile
index 28e38446..ff77351b 100644
--- a/.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile
+++ b/.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile
@@ -194,7 +194,8 @@ pipeline {
                     "eck-79-${BUILD_NUMBER}-e2e",
                     "eck-710-${BUILD_NUMBER}-e2e",
                     "eck-711-${BUILD_NUMBER}-e2e",
-                    "eck-712-${BUILD_NUMBER}-e2e"
+                    "eck-712-${BUILD_NUMBER}-e2e",
+                    "eck-713-${BUILD_NUMBER}-e2e"
```

Reply:
👍